person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  def initialize(_id, name, _age)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = 10
+  end
+  attr_reader :id
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] The `of_age?` method should be private, kindly consider moving the method down and adding `private` so the app is private